src/jdk.jartool/share/man/jar.md

```diff
@@ -213,6 +217,26 @@ operation modes:
 `--version`
 :   Prints the program version.
 
+## Integrity of a jar Archive
+As a jar archive is based on ZIP format, it is possible to create a jar archive using tools
+other than the `jar` command. The --validate option may be used to perform the following
+integrity checks against a jar archive:
```

Comment:
Nit - "integrity checks against a JAR file:"

Reply:
I think we have other inconsistencies that we can clean up at a later time in the help and man page for the jar tool in the usages of archive/file.

The man page description can use a further overhaul as we still reference applets.  So we can do this also as follow on work

Reply:
I changed references in the new content and leave the retrofit to future works.